### PR TITLE
Update ttrss version

### PR DESCRIPTION
**Pull request by rndmh3ro**
Thix PR updates the sources to the latest tt-rss version.

It also adds the database update to the install script. This is necessary because tt-rss runs a database migration after first install.

Also this PR is huge, sorry for that.

### CODE REVIEW

**Comment by M5oul** [comment]
Thanks for this work.
[App packaging team changed the policy to manage upstream sources retrieval](https://dev.yunohost.org/issues/201#Conclusion-chart).
Please download sources from upstream project from install and upgrade scripts.

**Comment by rndmh3ro** [comment]
That's good to hear, I didn't really like including the current installation-way anyway.

I changed the installation method to download the source-zip from tt-rss.org and install this one (much like the gogs-ynh app).

**Comment by M5oul** [comment]
Nice thanks.
It's not easy to review the modification from GitHub as the diff of this commit is very huge.
May you split in two commits remove source and modifications on other scripts.
I will let other people interested/using ttrss review this PR as I have other package to maintain.

**Comment by rndmh3ro** [comment]
Good ieda, done!

File: scripts/_common.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#
+# Common variables
+#
+
+APPNAME="ttrss"
+
+# ttrss version
+VERSION="16.8"
```
Comment (M5oul) [inline scripts/_common.sh]: Why it's not the same version than on the manifest?

Comment (rndmh3ro) [inline scripts/_common.sh]: Fixed!

**Comment by M5oul** [comment]
Ok, it seems ok.
I didn't tested.
Does a fresh install and upgrade works fine?
Someone else will need to approve this change.
CI-Apps could also be passed on it.
Then, you will be able to [ask for ttrss upgrade on offical list](https://github.com/YunoHost/apps/).

**Comment by rndmh3ro** [comment]
> Does a fresh install and upgrade works fine?
Yes it does. I tested it.

**Pull request merged**